### DISCUSSION TITLE
Drop down menues issue reappeared

### DISCUSSION THREAD

**Discussion by StSchwerdtfeger** [knowledgefutures/pubpub#3181] [Support]
A couple of month ago, we have experienced an issue on our pubpub website before where drop down menues in the banner didn't work anymore (i.e. drop down menues don't appear anymore). When clicking on pubs the title of the pub was also weirdly highlighted as if marked or so. The issues reappeared and are again present in both phone (android) and PC. Version of firefox (didn't check other browsers, but last time it was also present in edge). This is our website: https://journal.medicine.berlinexchange.de/ The last time I tried a lot such as deleting cookies etc. didn't work. This time they start working again when viewing an actual pub not a site/page (and being logged in). Thanks for your help in advance!

**Comment by eastofesten** [comment]
Hello! I cleared the cache & the dropdown menus are working again (tested in Firefox.) If it happens again, try making a change to the [Community Settings](https://journal.medicine.berlinexchange.de/dash/settings/details) (like adding or deleting a space in the Description.)
